developers/weaviate/manage-data/read-all-objects.mdx

```diff
@@ -109,7 +109,7 @@ Read through all data including `vector embeddings`.
   </TabItem>
 </Tabs>
 
-## Read all objects with Tenants
+## Read all objects with tenants
```

Comment:
Consider
Read all objects that have tenants

At the moment it sounds like we are using tenants to read objects.

Reply:
Updated